samples/SkijaInjectSample/src/main/kotlin/SkijaInjectSample/App.kt

```diff
@@ -56,8 +74,41 @@ fun createWindow(title: String) = runBlocking(Dispatchers.Swing) {
         }
     })
 
+    val defaultScreenshotPath = "/tmp/compose.png"
```

Comment:
Better to use crossplatform path `System.getProperty(“user.home”)` (or `Files.createTempFile()`)